src/components/bar.module.scss

```diff
@@ -27,7 +28,7 @@
 }
 
 @media screen and (max-width: 760px) {
-  .barContainer {
-    flex: 0 0 50%;
-  }
+  // .barContainer {
+  //   flex: 0 0 50%;
+  // }
```

Comment:
Did you want to keep this around?

Reply:
I fixed that in my current branch; will be in future PR